crates/api/src/local_user/reset_password.rs

```diff
@@ -6,23 +6,29 @@ use lemmy_api_common::{
   SuccessResponse,
 };
 use lemmy_db_views::structs::{LocalUserView, SiteView};
-use lemmy_utils::error::{LemmyErrorExt, LemmyErrorType, LemmyResult};
+use lemmy_utils::error::LemmyResult;
+use tracing::error;
 
 #[tracing::instrument(skip(context))]
 pub async fn reset_password(
   data: Json<PasswordReset>,
   context: Data<LemmyContext>,
 ) -> LemmyResult<Json<SuccessResponse>> {
-  // Fetch that email
   let email = data.email.to_lowercase();
-  let local_user_view = LocalUserView::find_by_email(&mut context.pool(), &email)
-    .await
-    .with_lemmy_type(LemmyErrorType::IncorrectLogin)?;
+  let _ = try_reset_password(&email, &context).await;
+  Ok(Json(SuccessResponse::default()))
+}
 
+async fn try_reset_password(email: &str, context: &LemmyContext) -> LemmyResult<()> {
```

Comment:
Should be moved to utils like all the other helper functions.

Reply:
I think it's a little better to keep the `reset_password` and `try_reset_password` functions together